### ISSUE TITLE
Cannot allocate 1318267840 bytes of memory

### ISSUE THREAD

**Issue by acravenho** [poanetwork/blockscout#662]
With the large number of deferred processes (internal transactions, balances, and tokens) we have run out of memory on Ethereum Mainnet deployment causing the Erlang VM to crash. 

After a certain number of deferred processes, we should start deleting them from Memory in order to stop the crashes. 

### Environment

* Operating System: Linux

### Steps to reproduce

1. Deploy Ethereum Mainnnet
2. View `erl_crash.dump` file after 12-14 hours

### Expected behaviour

Not crash.

### Actual behaviour

Crashes

**Comment by KronicDeth** [comment]
Is this still an issue after #707 fixed the memory leak in BuffedTask?

**Comment by acravenho** [comment]
I don't think this is still an issue. I will close this for now and keep an eye on the Ethereum Mainnet deployment. :+1:
